front-end/src/components/Signup/index.js

```diff
@@ -1,27 +1,53 @@
 import React from 'react';
 import styled from 'styled-components';
 import FormInput from '../../shared/FormInput';
+import { withRouter, Link } from "react-router-dom";
 import Logo from '../../svg/Logo';
+import axios from 'axios';
 
 class Signup extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      /*phoneNumber: '',
-      name: '',
-      username: '',*/
       email: '',
       password: '',
 
     };
   }
 
+  handleFormSubmit =async (event) =>{
+    event.preventDefault();
+    let payload={
+      "email":this.state.email,
+      "password":this.state.password
+    }
+
+    try{
+      const config = {
+        headers: {
+          'Content-Type': 'application/json'
+        }
+      }
+
+      const res = await axios.post('http://localhost:3000/users', payload, config);
```

Comment:
Can we move this logic into the api folder, for example, a function called signup which takes in payload and config as a parameter